### PR TITLE
trivial spelling correction

### PR DESCRIPTION
**Pull request by kwijibo**
s/Bare with me/Bear with me/

### CODE REVIEW

File: ch8.md [hunk 1]
```diff
@@ -526,7 +526,7 @@ Take a moment to consider how linear the control flow is here. We just read bott
 
 Goodness, would you look at that, `Task` has also swallowed up `Either`! It must do so in order to handle futuristic failures since our normal control flow does not apply in the async world. This is all well and good as it provides sufficient and pure error handling out of the box.
 
-Even with `Task`, our `IO` and `Either` functors are not out of a job. Bare with me on a quick example that leans toward the more complex and hypothetical side, but is useful for illustrative purposes.
+Even with `Task`, our `IO` and `Either` functors are not out of a job. Bear with me on a quick example that leans toward the more complex and hypothetical side, but is useful for illustrative purposes.
```
Comment (hemanth) [inline ch8.md]: :bear:

**Comment by DrBoolean** [comment]
Haha, lovely. Thanks!

**Comment by kwijibo** [comment]
Love the book, looking forward to the next chapter.
I'd also love to read something like a style guide for FP in js, with tips on keeping all those composed, curried, and partially applied functions readable and maintainable, good naming conventions for partially applied functions, etc

**Comment by DrBoolean** [comment]
Thanks! A style guide would be cool. All I've got so far is `variable_name` and `functionName` to distinguish first class.

**Comment by kwijibo** [comment]
The haskell style type signatures you describe are also useful; I wonder if there is a case for a version more idiomatic to javascript based on fat arrows?

**Comment by DrBoolean** [comment]
There's definitely a lot of room for creativity and JavaScript specific tweaks. I think copying haskell is just a good start. @naomik is doing some really cool stuff with es6. I really enjoyed this gist: https://gist.github.com/naomik/a6658c8c26425d154b6f

**Pull request merged**